### PR TITLE
Deere rework styling

### PR DESCRIPTION
**Pull request by ronso0**

### CODE REVIEW

**Comment by Be-ing** [comment]
I don't really like using "o" to indicate an unchecked option. It is easier to distinguish unchecked and checked options that way, but changing the shape between that and the checked box feels odd. Any other ideas?

**Comment by Be-ing** [comment]
Maybe increasing the font size of the checkboxes would help?

**Comment by ronso0** [comment]
Hm, I think the font size of the button text is okay. Just increasing
the boxes and checkmarks would mean to introduce another sub-container
for them, wouldn't it? Something like <span> would be cool.

But I'm also looking for a proper solution to that for FlatNite and I'm
not happy with the unicode characters, so I'll vectorize boxes and
checkmarks and use them as left-aligned background for displayValue 0 & 1.
Like this one https://d30y9cdsu7xlg0.cloudfront.net/png/835-200.png

Maybe empty/filled square also works.

**Comment by ronso0** [comment]
&#9899; checked
&#9898; unchecked
This looks pretty clear in combination with brighter color for checked options
screenshot - 25 01 2017 - 14 30 01

**Comment by Be-ing** [comment]
This is better, but it might be with a non-interactive bulleted list. Maybe turn them into squares? Maybe make the checked icon a square with a prominent "X" in it?

**Comment by ronso0** [comment]
> This is better, but it might be with a non-interactive bulleted list.

You mean that it might be mistaken as radio list?

Squares
&#9746; checked `&#9746;`
&#9744; unchecked `&#9744;` (original)
Though I find the circles much more pleasant, there are so many squares in the skin already. And they have more contrast between un/checked. Up to you!

File: res/skins/Deere/deck_mixer_controls_col1_left.xml [hunk 1]
```diff
@@ -64,5 +64,9 @@
       </Template>
 
     </Children>
+    <Connection>
+      <ConfigKey>[Master],show_eqs</ConfigKey>
+      <BindProperty>visible</BindProperty>
+    </Connection>
```
Comment (ronso0) [inline res/skins/Deere/deck_mixer_controls_col1_left.xml:70]: moved this here to make code more clear in parent template

File: res/skins/Deere/left_deck_mixer_controls.xml [hunk 2]
```diff
@@ -14,19 +14,16 @@
...
+              <Template src="skin:deck_mixer_controls_col2.xml"/>
+              <Template src="skin:deck_mixer_controls_col3.xml"/>
             </Children>
-            <Connection>
-              <ConfigKey>[Master],show_eqs</ConfigKey>
-              <BindProperty>visible</BindProperty>
-            </Connection>
           </WidgetGroup>
-          <Template src="skin:deck_mixer_controls_col2.xml"/>
-          <Template src="skin:deck_mixer_controls_col3.xml"/>
```
Comment (ronso0) [inline res/skins/Deere/left_deck_mixer_controls.xml:29]: Mixer columns2/3 were squeezed and set off to the left, column1 was half-covered. This happened for Channel1 mixer only. Don't know why THIS works..
Reproduce:
- 2 decks
- split waveforms
=> with skin settings turn EQs off, then on again

File: res/skins/Deere/style.qss [hunk 3]
```diff
@@ -823,6 +823,7 @@ WWidget, QLabel {
   border: none;
   qproperty-alignment: 'AlignHCenter';
   font: 10px;
+  margin-top: -2px;
 }
```
Comment (ronso0) [inline res/skins/Deere/style.qss:827]: Label for EQs and other knobs was cut off.

**Comment by Be-ing** [comment]
LGTM, thanks.

**Pull request merged**